src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -77,9 +80,15 @@ namespace conduit { namespace blueprint { namespace mesh {
     static const std::vector<std::string> coord_systems(coord_system_list,
         coord_system_list + sizeof(coord_system_list) / sizeof(coord_system_list[0]));
 
-    static const std::string topo_type_list[4] = {"uniform", "rectilinear", "structured", "unstructured"};
+    static const std::string topo_type_list[5] = {"points",
+                                                  "uniform",
+                                                  "rectilinear",
+                                                  "structured",
+                                                  "unstructured"};
+
     static const std::vector<std::string> topo_types(topo_type_list,
```

Comment:
@xjrc I think I tackled the only change needed for verify. 

Verify is being tested as part of the blueprint mesh examples tests -- if this wasn't working I would expect a failure.

Reply:
This change is certainly a big part of making `conduit::blueprint::verify` work, but there are a few other minor changes that should be included. In particular, the test cases should be extended to test this case explicitly (see [`src/tests/blueprint/t_blueprint_mesh_verify.cpp:637`](https://github.com/LLNL/conduit/blob/master/src/tests/blueprint/t_blueprint_mesh_verify.cpp#L637) for details). Also, it may be worth adding a `conduit::blueprint::mesh::topology::points::verify` function as well to go along with all the other per-type verify functions.

Reply:
perfect, thanks for keeping all of this in mind!